source/content.js

```diff
@@ -25,6 +25,17 @@ function onNewTweets(cb) {
 	observeEl('#stream-items-id', cb);
 }
 
+function addLikeButtonNavbar() {
+	$('#global-actions').append(`
+		<li>
+			<a role="button" data-nav="favorites" href="/i/likes" class="js-nav js-tooltip js-dynamic-tooltip" data-placement="bottom" data-original-title="">
```

Comment:
I don't think you need to add `data-original-title=""`. It's added by the JS on hover.